Applications/xia2_main.py

```diff
@@ -63,6 +65,22 @@ def check_environment():
         )
 
 
+def check_hdf5_master_files(master_files):
```

Comment:
Please could you add a test for this function? As a standalone function, without any dependencies on global singletons, it should be relatively straightforward to add a test for this - both correctly given `master.h5` as input, and given data `.h5` files as input.

Reply:
Done